.travis.yml

```diff
@@ -18,7 +20,7 @@ notifications:
   email: false
 
 before_cache:
```

Comment:
btw, I've just figured that it doesn't work because we run in Docker, we can mount cache into container tho

Reply:
Nah, seems useless anyway. We spend too much time on spinning the Docker container, dependencies downloading is the least issue there is. Should I remove cache-related things?

Reply:
Well, I'm just afraid that mavenCentral/jcenter outage (and they do happen fairly often, especially with jcenter) will bring down CI

Reply:
The same can be applied to Docker Hub but we use it anyway.

Reply:
Sure, but I have not seen Docker hub down this year while jcenter has outages almost every month

But anyway, let's remove it at least for now since it does nothing :)

Reply:
Done!